code_generator/GdkCodeGenerator/Templates/UnityCommandPayloadGenerator.tt

```diff
@@ -17,92 +18,87 @@ namespace <#= qualifiedNamespace #>
 <# foreach (var commandDetails in commandDetailsList) {#>
         public class <#= commandDetails.CommandName #>
         {
-            public struct Request : IIncomingCommandRequest
+            public struct Request
             {
-                public uint RequestId { get; }
-
-                internal Translation Translation { get; }
+                public EntityId TargetEntityId { get; }
+                public <#= commandDetails.RequestType #> RawRequest { get; }
```

Comment:
Thing to consider for later - call it "Payload"? Calling something inside Request "RawRequest" seems a bit odd.

Reply:
Agreed - this was the existing nomenclature